src/Altinn.Common/Altinn.Common.PEP/Altinn.Common.PEP/Authorization/AppAccessHandler.cs

```diff
@@ -59,18 +59,18 @@ protected override async Task HandleRequirementAsync(AuthorizationHandlerContext
                 return;
             }
 
-            XacmlJsonRequestRoot request = DecisionHelper.CreateXacmlJsonRequestRoot(context, requirement, _httpContextAccessor.HttpContext.GetRouteData());
+            XacmlJsonRequestRoot request = DecisionHelper.CreateXacmlJsonRequest(context, requirement, _httpContextAccessor.HttpContext.GetRouteData());
 
             _logger.LogInformation($"// Altinn PEP // AppAccessHandler // Request sent: {JsonConvert.SerializeObject(request)}");
 
             XacmlJsonResponse response = await _pdp.GetDecisionForRequest(request);
 
-            if (response?.Response == null)
+            if (response == null || response?.Response == null)
```

Comment:
response?.Response == null is the same as response == null || response.Response == null